.github/workflows/daily-ci.yaml

```diff
@@ -48,102 +50,80 @@ jobs:
         run: |
           docker buildx build --platform linux/amd64,linux/arm64 .
 
-  build-on-ubuntu-without-jemalloc:
+  build-and-test:
+    name: Build And Test
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-18.04]
-    
-    runs-on: ${{ matrix.os }}
-    steps:
-      - name: Checkout Code Base
-        uses: actions/checkout@v2.3.4
-        with:
-          fetch-depth: 64
-
-      - name: Build
-        run: |
-          mkdir build && cd build
-          cmake -DDISABLE_JEMALLOC=true -DCMAKE_BUILD_TYPE=Release ..
-          make -j4
-          cd ..
-
-      - name: Redis Tcl Test
-        run: |
-          sudo apt-get install tcl8.5
-          cd tests/tcl && sh runtest && cd -
+        include:
+          - name: Ubuntu GCC ASan
+            os: ubuntu-18.04
+            with_sanitizer: -DENABLE_ASAN=ON
+          # - name: Ubuntu GCC TSan
+          #   os: ubuntu-18.04
+          #   with_sanitizer: -DENABLE_TSAN=ON
+          - name: Ubuntu GCC without Jemalloc
+            os: ubuntu-18.04
+            without_jemalloc: -DDISABLE_JEMALLOC=ON
+          - name: Ubuntu Clang ASan
+            os: ubuntu-18.04
+            with_sanitizer: -DENABLE_ASAN=ON
+            clang: -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
+          # - name: Ubuntu Clang TSan
```

Comment:
Why we comment out these lines?

Reply:
Refer to https://github.com/apache/incubator-kvrocks/pull/630#issuecomment-1158446352, we temporarily remove these job since it will fail due to some data race in kvrocks, and it will be re-add after the bug is fixed.

Reply:
OK. Then we should open an issue once this PR merged.